ansible/examples/slave.yml

```diff
@@ -627,6 +631,29 @@
             line: '%dist .el{{ ansible_distribution_major_version }}'
       when: ansible_os_family == "RedHat"
 
+    ## tmpwatch/tmpreaper TASKS
+    - name: tmpwatch/tmpreaper Tasks
+      block:
+        # In case we're running 'ansible-playbook --tags tmp'
```

Comment:
This is the thing that installs the package, tag or not, right?

Reply:
This installs the package in case

1. We're not running the whole playbook
2. The package is not installed and needs to be

Reply:
The package would get installed during `## PACKAGE INSTALLATION TASKS` but not if we're running `ansible-playbook slave.yml --tags tmp`